tests/e2e/cluster_creation.go

```diff
@@ -50,7 +51,7 @@ var _ = Describe("TF Test", func() {
 
 			By("Create account-roles")
 			accRoleParam := &EXE.AccountRolesArgs{
-				Token:             token,
+				Token:             os.Getenv(CON.TokenENVName),
```

Comment:
I would take it once and save as constant/var

Reply:
ack, changed to:
```
var (
	TokenENVName = os.Getenv("RHCS_TOKEN")
)
```